examples/ssh/iperf3.js

```diff
@@ -101,7 +122,26 @@ function validate() {
 * @documentation This procedure is used for retrieving iperf3 results
 */
 function get_status() {
```

Comment:
This does not capture error cases (server not responding)

iperf3 -f m -c ping-90ms.online.net -p 5209 -R
iperf3: error - unable to connect to server: Connection refused